api/profile/profileModel.js

```diff
@@ -24,6 +24,18 @@ const update = (id, profile) => {
     .returning('*');
 };
 
+const updateIsActive = (profile_id, status) => {
```

Comment:
Thank you so much for setting this up, you cleared the way for us on the front end!